web/src/components/storage/PartitionPage.tsx

```diff
@@ -491,10 +499,18 @@ type PartitionDescriptionProps = {
 };
 
 function PartitionDescription({ partition }: PartitionDescriptionProps): React.ReactNode {
+  const label = partition.filesystem?.label;
+
   return (
     <Split hasGutter>
-      <span>{partition.description}</span>
-      <span>{deviceSize(partition.size)}</span>
+      <SplitItem>{partition.description}</SplitItem>
+      {label && (
+        <SplitItem>
+          <Label isCompact variant="outline">
+            {label}
+          </Label>
+        </SplitItem>
```

Comment:
What about still using the Flex with custom size approach?

Reply:
Well, I like the separation given by Split here.

Reply:
So, go ahead. 

As commented in IRC, it should not be a question of taste but consistence and correct abstractions to improve maintenance and those things that we use to make so important in the backend side :wink: